fastestimator/backend/feed_forward.py

```diff
@@ -15,17 +15,22 @@
 from typing import TypeVar, Union
 
 import tensorflow as tf
-
 import torch
 
+from fastestimator.backend.to_tensor import to_tensor
+
 Tensor = TypeVar('Tensor', tf.Tensor, torch.Tensor)
 
 
 def feed_forward(model: Union[tf.keras.Model, torch.nn.Module], x: Tensor, training: bool = True) -> Tensor:
     if isinstance(model, tf.keras.Model):
+        if not isinstance(x, tf.Tensor):
```

Comment:
why are we doing this here? will add overhead to all training

Reply:
moreover, if we're allowing user to pass something that isn't a Tensor as 'x', we need to update the typing

Reply:
(but again, we shouldn't do that)

Reply:
it won't affect the training speed from my benchmark.    My intent is to enable `feed_forward` for model prediction.

Reply:
fixed the typing